Autopilot/AttitudeManager/GetFromPathManager.cpp

```diff
@@ -4,20 +4,28 @@
  * Code
  **********************************************************************************************************************/
 
-
-#ifndef SIMULATION
-
 // dummy implementation until real one is required.
 PMError_t PM_GetCommands(PMCommands *Commands)
 {
-	(void) Commands;
+#ifdef SIMULATION
+// Commands are hard coded until the path manager state machine is written and can communicate with attitude manager.
+
 
+	Commands->roll = 0;
+	Commands->pitch = 0;
+	Commands->yaw = 0;
+	Commands->airspeed = 60;
 	PMError_t errorStruct;
 	errorStruct.errorCode = 0;
 
 	return errorStruct;
-}
-
 #else
+	(void) Commands;
 
+	PMError_t errorStruct;
+	errorStruct.errorCode = 0;
+
+	return errorStruct;
 #endif
```

Comment:
yea you're right, it's not all that pretty. What I might do instead is produce a separate cpp file and link that into the Simulation build instead of this one.